contents/security/sqlInjection.md

```diff
@@ -0,0 +1,145 @@
+# SQL Injection
+
+Author: [Lewis Koh](https://github.com/nus-cs3281/2018/blob/master/students/lewisKoh/lewisKoh-Resume.md)
+
+## Overview
+
+SQL (Structured Query Language) Injection is a vulnerability found in websites using databases.
+
```

Comment:
I think probably you can introduce the sql query language a bit, since the injection makes use of the query language structure and how the query is parsed